### DISCUSSION TITLE
Why Redis can not be recognized as a Cache Service?

### DISCUSSION THREAD

**Discussion by holtenko** [apache/skywalking#8596] [Q&A] [answered]
It's my first time to use Skywalking.
I use Redis(Redisson) in my project , and I can find the Redisson records in the Trace page.
But when I check the ES data, I can only find the service itself and MySQL as Database service.
I don't know why Redis can not be recognized as a Cache Service?

**Comment by wu-sheng** [comment, marked as answer]
I have to say there is no reason. Because no contributor said `would like to create a dashboard to cache`, then nothing happened. This is open source.

**Reply by holtenko** [reply]
> I have to say there is no reason. Because no contributor said `would like to create a dashboard to cache`, then nothing happened. This is open source.

You mean it's not implemented yet, and I can create PR for this?

**Reply by wu-sheng** [reply]
It is not implemented. If you want to contribute, you should propose a design here about what you are going to add and change.
I created the database dashboard before, it is not just UI side work, a lot of changes have to be done in the OAP server. So, we need your design, and several PRs, which you need to do step by step.

**Reply by holtenko** [reply]
Yes, thanks for your quick reply. I think I have to spend some time in reading the source code. Then, I will try to contribute for this. Thank you again,

**Reply by wu-sheng** [reply]
Take your time.

Notice, UI is moving to 3rd version soon, https://github.com/apache/skywalking-booster-ui
Don't waste time on Rocketbot UI, but focus on backend first. Most v9 backend codes and all v9 APIs are ready.
